scripts/aws/monitor.py

```diff
@@ -179,6 +185,22 @@ def get_topic_arn(client, topic_name):
     return None
 
 
+def enable_cloudwatchlogs_in_stages(client, gw_name, logging_level):
+    """enables the cloudwatch logs for all the stages"""
+    gw_res = client.get_rest_apis()
+    for value in gw_res['items']:
+        if value['name'] == gw_name:
```

Comment:
Ya makes sense. I can have user pass in the gateway ID to the script as an argument. Do you think thats acceptable?